packages/reports/addon/consumers/request/filter.ts

```diff
@@ -117,7 +117,9 @@ export default class FilterConsumer extends ActionConsumer {
       const { request } = route.modelFor(routeName) as ReportModel;
 
       const findDefaultOperator = (type: string) => {
+        type = type.toLowerCase();
         const opDictionary: Record<string, string> = {
```

Comment:
It would be nice to type this record with `valueType`s and filter operator types